custom-namespaces.yaml

```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+moz_org_sites:
+  glean_app: false
+  pretty_name: Mozilla Org Websites
+  spoke: looker-spoke-default
+  owners:
+    - kignasiak@mozilla.com
+  views:
+    moz_org_site_page_metrics:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_page_metrics_v1
+    moz_org_site_metrics_summary:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_metrics_summary_v1
+    moz_org_site_landing_page_metrics:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_landing_page_metrics_v1
+    moz_org_blogs_daily_summary:
```

Comment:
Leave this as `blogs_daily_summary` blogs = blogs.mozilla.org and how people reference it internally so should have different naming than moz org = mozilla.org